diff --git a/gopkg.in/lxc/go-lxc.v2/container.go b/gopkg.in/lxc/go-lxc.v2/container.go
index 4d63144..132be18 100644
--- a/gopkg.in/lxc/go-lxc.v2/container.go
+++ b/gopkg.in/lxc/go-lxc.v2/container.go
@@ -361,12 +361,12 @@ func (c *Container) Create(options TemplateOptions) error {
 	}
 
 	// unprivileged users are only allowed to use "download" template
-	if os.Geteuid() != 0 && options.Template != "download" {
+	if os.Geteuid() != 0 && options.Template != "download" && !strings.HasSuffix(options.Template, "lxc-lm-download") {
 		return ErrTemplateNotAllowed
 	}
 
 	var args []string
-	if options.Template == "download" {
+	if options.Template == "download" || strings.HasSuffix(options.Template, "lxc-lm-download") {
 		// required parameters
 		if options.Distro == "" || options.Release == "" || options.Arch == "" {
 			return ErrInsufficientNumberOfArguments
